src/table/utils/__tests__/handle-accessiblity.spec.js

```diff
@@ -137,16 +130,18 @@ describe('handle-accessibility', () => {
 
     it('should indirectly call setFocusedCellCoord with adjusted index, and keyboard.focus', () => {
       handleAccessibility.handleClickToFocusBody(cellData, rootElement, setFocusedCellCoord, keyboard);
-      expect(cell.setAttribute).have.been.calledOnceWith('tabIndex', '-1');
-      expect(setFocusedCellCoord).to.have.been.calledOnceWith([1, 0]);
-      expect(keyboard.focus).to.have.been.calledOnce;
+      expect(cell.setAttribute).toHaveBeenCalledTimes(1);
+      expect(cell.setAttribute).toHaveBeenCalledWith('tabIndex', '-1');
+      expect(setFocusedCellCoord).toHaveBeenCalledWith([1, 0]);
```

Comment:
To be consistent you should check `oHaveBeenCalledTimes(1)` on both functions, or none of them. Could they possibly be called more than once? If not checking with `toHaveBeenCalledWith` should be enough